src/clj/rems/css/styles.clj

```diff
@@ -350,15 +356,17 @@
     [:label.form-check-label {:cursor :pointer}]]
    [:html {:position :relative
            :min-width (u/px 320)
+           :min-height "100vh"
```

Comment:
What does this do?

Reply:
it makes sure that footer is not lifter above the end of screen

Reply:
Why is it needed in this PR specifically? What has changed that has broken the page? To me it seems like everything works fine even without so in what case does it not work?

Reply:
I sometimes would see the footer slightly lifted so I thought It would not hurt to add it
But it can be a separate change if this is important

Reply:
so, would you like me to remove it?

Reply:
Can you reproduce it without this? If you can then please let me know how. If it is random are you sure this fixes it?

Reply:
Ok, ill try!

Reply:
yes, I did